visual-studio-dev-vm-chocolatey/SetupChocolatey.ps1

```diff
@@ -0,0 +1,44 @@
+param([Parameter(Mandatory=$true)][string]$chocoPackages)
+
+cls
+# Grab the choco installation script
+iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1'))
+
+# Get username/password & machine name
+$userName = "artifactInstaller"
+[Reflection.Assembly]::LoadWithPartialName("System.Web") 
+$password = $([System.Web.Security.Membership]::GeneratePassword(12,4))
+$cn = [ADSI]"WinNT://$env:ComputerName"
+
+# Create new user
```

Comment:
This is where the bulk of the dev/testing was done.  Normal admin user does not work - chocolatey installations hang.